packages/@cdktf/hcl2cdk/lib/generation.ts

```diff
@@ -416,8 +438,12 @@ function mapConfigPerResourceType(resource: string, item: Resource[0]) {
   return item;
 }
 
+const loopComment = `In most cases loops should be handled in the programming language context and 
+not inside of the Terraform context. If you are looping over something external, e.g. a variable or a file input
```

Comment:
The word "variable" is a bit too ambiguous 😅 I'd expect most practitioners using convert to think if "Terraform Variables" which would require iterators – Can we make this more clear somehow?

Reply:
Let's change the text in a different PR, I just moved it but is has been like this before. We can definitely change it but it's not in the scope of this PR.